standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -128,6 +128,13 @@ public class MetastoreConf {
       keyToVars.put(var.varname, var);
       keyToVars.put(var.hiveName, var);
     }
+    Configuration.addDeprecations(new Configuration.DeprecationDelta[] {
```

Comment:
can we just drop them in Hive-4.1?

Reply:
Ideally, they should have been dropped then in 4.0 release.

Reply:
Since the work/effort is done and the tests are green there is no downside in merging this PR as is. I will create a follow-up to remove these props completely.

Reply:
Created follow-up: https://github.com/apache/hive/pull/5931